docker-compose.override.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+
+services:
+  firefly-db:
+    volumes:
+      - firefly-dev-storage:/var/lib/mysql
+      - .:/var/www/firefly-iii
```

Comment:
That's actually handled by the [Dockerfile](https://github.com/firefly-iii/firefly-iii/blob/develop/Dockerfile#L45)